webapp/channels/src/components/announcement_bar/show_start_trial_modal/show_start_trial_modal.tsx

```diff
@@ -74,7 +76,7 @@ const ShowStartTrialModal = () => {
         const now = new Date().getTime();
         const hasEnvMoreThan6Hours = now > installationDatePlus6Hours;
         const hasEnvMoreThan10Users = Number(totalUsers) > userThreshold;
-        if (isUserAdmin && !isBenefitsModalOpened && hasEnvMoreThan10Users && hasEnvMoreThan6Hours && !hadAdminDismissedModal && !isLicensedOrPreviousLicensed) {
+        if (isUserAdmin && !isMySql && !isBenefitsModalOpened && hasEnvMoreThan10Users && hasEnvMoreThan6Hours && !hadAdminDismissedModal && !isLicensedOrPreviousLicensed) {
```

Comment:
I had even forgetten we had this trigger to show the start trial modal. Given that we wont cover all cases in the front end, we should have the api request do the check so it just returns an error for mysql in case it is requested from a spot we are not covering: 

https://github.com/mattermost/mattermost/blob/e02e1aab102b9825496994a24034d62b4573eebf/server/channels/api4/license.go#L179

Reply:
Yes, it will probably fail (unless it's loading somewhere else we missed) but if we can avoid an extra request out to the portal as we know it will fail then even better. 0/5

Reply:
Sounds good, thank you! Done :)